libc/startup/linux/aarch64/CMakeLists.txt

```diff
@@ -1,17 +1,13 @@
 add_startup_object(
-  crt1
+  start
```

Comment:
why is this target being renamed? Additionally, if you rename it here it also needs to be renamed for your `merge_relocatable_object`.

Reply:
I have renamed it in `merge_relocatable_object`. I did this because it is no longer `crt1.o` but a component of it.